src/System.Security.Cryptography.Encoding/tests/Asn1/Reader/ReadT61String.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security.Cryptography.Asn1;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Tests.Asn1
+{
+    public sealed class ReadT61String : Asn1ReaderTests
+    {
+        public static IEnumerable<object[]> ValidEncodingData { get; } =
+            new object[][]
+            {
+                // https://github.com/dotnet/corefx/issues/27466
+                new object[]
+                {
+                    PublicEncodingRules.DER,
+                    "140E47726170654369747920696E632E",
+                    "GrapeCity inc.",
+                },
+                new object[]
+                {
+                    PublicEncodingRules.DER,
+                    "1411546F6F6C7320446576656C6F706D656E74",
+                    "Tools Development",
+                },
+                // Mono test case taken from old bug report
+                new object[]
+                {
+                    PublicEncodingRules.DER,
+                    "14244865646562792773204DF862656C68616E64656C202F2F204356523A3133343731393637",
+                    "Hedeby's M\u00f8belhandel // CVR:13471967",
+                },
+                new object[]
+                {
+                    PublicEncodingRules.DER,
+                    "14264865646562792773204DF862656C68616E64656C202D2053616C6773616664656C696E67656E",
+                    "Hedeby's M\u00f8belhandel - Salgsafdelingen",
+                },
+                // Valid UTF-8 string is interpreted as UTF-8
+                new object[]
+                {
+                    PublicEncodingRules.DER,
+                    "1402C2A2",
+                    "\u00a2",
```

Comment:
There are enough cases in the other tests to know that multiple-segment reconstruction works just fine, but can you copy this test case to a BER one where it splits the segments in the middle of the UTF-8 character, just to make me happy?

`34800401C20401A20000` looks like it would do it.

Reply:
Alright, I will do it tomorrow (too late in my time zone now).